justfile

```diff
@@ -59,14 +59,14 @@ workspace-optimize:
             --mount type=volume,source="$(basename "$(pwd)")_cache",target=/target \
             --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
             --platform linux/arm64 \
-            cosmwasm/workspace-optimizer-arm64:0.14.0; \
+            cosmwasm/optimizer:0.16.0; \
```

Comment:
Shouldnt this be optimizer-arm64 as well?

Reply:
lol me dumb